frontend/ui/widget/touchmenu.lua

```diff
@@ -1018,14 +1016,7 @@ function TouchMenu:search(search_for)
             end
         end
 
-        if val.sub_item_table_func and extensive_search then
-            local sub_item_table = val.sub_item_table_func()
-            local perpage = ""
-            if sub_item_table.max_per_page then
-                perpage = "[" .. sub_item_table.max_per_page .."]"
-            end
-            recurse(sub_item_table, path .. ".sub_item_table_func" .. perpage, text, icon, depth)
-        elseif val.sub_item_table then
+        if val.sub_item_table then
```

Comment:
After a quick look, it seems that with extensive_search=false, it wasn't (and isn't) searching inside sub_item_table_func ?
(one example of menu item inside such a func: `About font-family fonts` or `Ignore publisher font names when font-family is set`)
If that's the case, put the above back with just `if val.sub_item_table_func then` ?